File: src/main/java/com/aquasecurity/plugins/trivy/model/Secret.java
package com.aquasecurity.plugins.trivy.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Secret{
    @JsonProperty("RuleID")
    public String ruleID;
    @JsonProperty("Category")
    public String category;
    @JsonProperty("Severity")
    public String severity;
    @JsonProperty("Title")
    public String title;
    @JsonProperty("StartLine")
    public int startLine;
    @JsonProperty("EndLine")
    public int endLine;
    @JsonProperty("Match")
    public String match;
}

File: src/main/kotlin/com/aquasecurity/plugins/trivy/model/report/Secret.kt
package com.aquasecurity.plugins.trivy.model.report

import com.fasterxml.jackson.annotation.JsonProperty

class Secret {
  @JsonProperty("RuleID") var ruleID: String? = null

  @JsonProperty("Category") var category: String? = null

  @JsonProperty("Severity") var severity: String? = null

  @JsonProperty("Title") var title: String? = null

  @JsonProperty("StartLine") var startLine: Int = 0

  @JsonProperty("EndLine") var endLine: Int = 0

  @JsonProperty("Match") var match: String? = null
}
